### PR TITLE
Turn on more pylint warnings

### PR DESCRIPTION
**Pull request by zenhack**
* No more `import *`
* Check for unused variables.

I plan on doing a couple of these a week going forward.

### CODE REVIEW

**Comment by zenhack** [comment]
I don't understand why the github page says travis is still in progress -- if you click on the link it says it's done & passed.

File: hil/client/node.py [hunk 1]
```diff
@@ -26,11 +26,11 @@ def register(self, node, subtype, *args):
         # Node requires which OBM, and knows arguments required
         # for successful node registration.
 
-        obm_api = "http://schema.massopencloud.org/haas/v0/obm/"
-        obm_types = ["ipmi", "mock"]
         # FIXME: In future obm_types should be dynamically fetched.
         # We need a new api call for querying available
         # and currently active drivers for HIL
+        # obm_api = "http://schema.massopencloud.org/haas/v0/obm/"
+        # obm_types = ["ipmi", "mock"]
```
Comment (naved001) [inline hil/client/node.py]: off topic note: I read this the whole comment here, and now we have an API to get active drivers, so I guess we can implement this now (in a separate PR of course).

**Comment by zenhack** [comment]
Quoting Naved Ansari (2017-08-10 14:34:35)

> -from hil import model
> +from hil import model, errors
>
>    if we import model here, then why are we importing db from hil.model
>    again in the next line?


We use the db variable often enough that it seemed a bit nicer to just
import that one directly. We *could* just do model.db.session...
everywhere; it's a question of style.

**Comment by naved001** [comment]
Cool. I am happy.

**Pull request merged**